srcpkgs/vscode/template

```diff
@@ -46,7 +45,7 @@ do_build() {
 	export NODE_OPTIONS="${_mem_limit}"
 	/usr/lib/node_modules/npm/bin/node-gyp-bin/node-gyp install \
 		--target=$_electronver \
-		--tarball=/usr/include/electron7/node_headers.tar.gz
+		--tarball=/usr/include/electron9/node_headers.tar.gz
```

Comment:
I think this could be programatically set, `"${_electronver%%.*}` i think

Reply:
Absolutely. Thx. Gonna update this ASAP.

Reply:
You could also only specify the major version as variable and read the full version from `/usr/lib/electron${electon_major}/version`